agenta-web/src/components/Playground/Views/ParametersCards.tsx

```diff
@@ -0,0 +1,282 @@
+import {Row, Card, Slider, Select, InputNumber, Col, Input, Button} from "antd"
+import React, {ParamHTMLAttributes} from "react"
+import {Parameter, InputParameter} from "@/lib/Types"
+import {renameVariables} from "@/lib/helpers/utils"
+import {useEffect} from "react"
+
+interface ModelParametersProps {
+    optParams: Parameter[]
+    onChange: (param: Parameter, value: number | string) => void
+    handleParamChange: (name: string, value: number | string) => void
+}
+
+export const ModelParameters: React.FC<ModelParametersProps> = ({
+    optParams,
+    onChange,
+    handleParamChange,
+}) => {
+    return (
+        <>
+            {optParams?.filter((param) => !param.input && param.type === "number").length > 0 && (
```

Comment:
`.some` is a better candidate here instead of `.filter`

Reply:
done